src/main/cpp/t07_quad2.cpp

```diff
@@ -20,6 +20,33 @@
 
 using namespace std;
 
-int t07_quad2() {
 
-};
\ No newline at end of file
+int t07_quad2(){
+    float Q, W, E;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'R' can be reduced.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4612954)